iFixit/pulldasher
# Consider draft pull requests to be dev_blocked

### Overview:

Previously, a pull request marked as a draft would stay in the **CR/QA** column. This pull request will adjust that by placing these _draft-state_ pull requests into the **Dev blocked** column.

![draft](https://user-images.githubusercontent.com/95656772/182685883-78c3d750-cf8c-49c1-ad96-5e6ce1bd9c8d.png)

> May want to add a sort to this eventually

### CR/QA Notes:

> **CR:** Review to https://github.com/iFixit/pulldasher/pull/329/commits/6063432395f411dd4fccaabfc7323d035f43563b

> **QA:**
>
> You can probably just skip to step 3.)
>
> 1. Checkout | merge with [`draft-dev-block`](https://github.com/iFixit/pulldasher/tree/draft-dev-block)
> 2. Run `/home/ifixit/ifixit_scripts/update-pulldasher-dev --run .`
> 3. Navigate to https://pulldasher-dev.cominor.com/
> 4. Ensure that draft pull requests have a flag and are placed in the **Dev blocked** column


---

cr_req 2

Closes #230
